Formula/terracognita.rb

```diff
@@ -1,8 +1,9 @@
 class Terracognita < Formula
   desc "Reads from existing Cloud Providers and generates Terraform code"
   homepage "https://github.com/cycloidio/terracognita"
-  url "https://github.com/cycloidio/terracognita/archive/v0.8.3.tar.gz"
-  sha256 "7713e4b93528c294db86d008a2ab09ed841984ae2d1ea6f58c3703dab3a1b21e"
+  url "https://github.com/cycloidio/terracognita/archive/v3.9.0-cy.tar.gz"
+  version "3.9.0-cy"
+  sha256 "84daa5ae580ed0d095d2bae2ef01160557cbeaa7c18469491073200bb6b72dd4"
```

Comment:
Big jump in version number; weird version scheme. Likely not a stable release.

Reply:
yeah, probably a wrong tag, raise as question into the upstream, https://github.com/cycloidio/terracognita/issues/385

Reply:
Looks like they removed that tag
![image](https://github.com/Homebrew/homebrew-core/assets/1580956/16c89f12-8249-4b03-a074-b7514a562334)